_layouts/product.html

```diff
@@ -18,12 +18,15 @@ <h1 class="margin-top-0">{{ page.title }}</h1>
         <span class="usa-tag bg-accent-warm-light text-base-darker display-inline-block">{{ item }}</span> 
         {% endfor %}
     </p>
-
+   
     <div class="grid-container padding-x-0">
         <div class="grid-row">
           <div class="tablet:grid-col tablet:padding-right-1">
             <img src="{{ page.feature-image }}" alt="{{ page.feature-image-description }}">
             <button class="usa-button usa-button--unstyled margin-y-2">View more images</button>
+            <div class="margin-y-2">
```

Comment:
We don't need the extra div here. You can apply the margin spacing class to the button element directly.

Reply:
When I remove the div, the button becomes in-line with the "view image" link. Is that okay? 
![Screen Shot 2021-05-14 at 12 51 01 PM](https://user-images.githubusercontent.com/52218695/118303280-441f9c00-b4b3-11eb-9ffb-967fde2cae7f.png)